MBBSEmu.Tests/CPU/IMUL_Tests.cs

```diff
@@ -0,0 +1,116 @@
+﻿using Iced.Intel;
+using Xunit;
+using static Iced.Intel.AssemblerRegisters;
+
+namespace MBBSEmu.Tests.CPU
+{
+    public class IMUL_Tests : CpuTestBase
+    {
+        [Theory]
+        [InlineData(1, -1, -1, false, false)]
+        [InlineData(-1, -1, 1, false, false)]
+        [InlineData(-127, -1, 127, false, false)]
+        [InlineData(127, -1, -127, false, false)]
```

Comment:
how about some regular values here, like your 5x10 example down below

+ the next two tests